### ISSUE TITLE
Photo and Audio Message are showing with current date only?

### ISSUE THREAD

**Issue by imran233113** [zhuozhuo/ZHChat#22]
Hi i am sending test message and send my date along with message, but what about photo and audio message, i could not pass date along with the photo and audio message, so it take always current date. 

How to fix that ?

**Comment by zhuozhuo** [comment]
Thank you for your feedback, picture loading is due to the image link for HTTP causes, has been repaired. As for recording sound, do you open the microphone permissions in the system settings?

**Comment by imran233113** [comment]
Hi Thanks for reply,

My admin is sending me image+message or Audio+message together, in that case messages are not displaying. 
I want to see image and message or audio with message.  Is it possible ?

**Comment by zhuozhuo** [comment]
This is achievable, you just follow the ZHCMessageMediaData protocol or inherit the ZHCMediaItem custom you need to MediaItem!

**Comment by imran233113** [comment]
HI, i could not get custom to add in ZHCMediaItem . could you update with code please

**Comment by zhuozhuo** [comment]
Yes, you can't customize `ZHCMediaItem` directly. You must write a class and inherit from `ZHCMediaItem`. You can refer to the `ZHPhotoItem` in the Demo.

**Comment by imran233113** [comment]
thanks i will check if any thing i will revert back to you.. thanks a lot

**Comment by imran233113** [comment]
Hi It will take time for me write the code of your level really. If you could update me with code how could i add image+test that would be good and thankful to you.

**Comment by zhuozhuo** [comment]
Sorry, in the busy other things! Temporarily can't spare time to help you!

**Comment by imran233113** [comment]
ok once you are free, please update it. thanks
